src/mains/unix.cpp

```diff
@@ -517,6 +520,8 @@ long    DisplayListOfPostChoices (void)
 void    ProcessConfigStr (_String& conf)
 {
     _String errMsg;
+    _String  verString (GetVersionString().getStr());
```

Comment:
Delete these two lines when you make the change below.

Reply:
Having attempted this change, here is what the compiler has to say:
```
/Users/sjspielman/hyphy/src/mains/unix.cpp:536:57: error: base operand of '->' has non-pointer type '_String'
             fprintf(stderr, "\n%s\n", GetVersionString()->getStr() );
                                                         ^~
make[3]: *** [CMakeFiles/HYPHYMP.dir/src/mains/unix.cpp.o] Error 1
```

Reply:
Oh yeah, do `GetVersionString().getStr()`